server/app/routes.py

```diff
@@ -233,3 +277,16 @@ def fetch_all_report_infos():
         return make_response('Server Error'), 500
 
     return make_response('suceess', MOSS_info), 200
+
+@app.route('/check/download', methods=['get'])
+def download_report():
+    file_name = ""
+    file_path = ""
+    try:
+        file_path, file_name = ChecksController.download_check_details(request.args)
+    except Exception as e:
+        print(e)
+        return make_response('Server Error'), 500
+
+    response = send_from_directory(directory=file_path, path=file_name, as_attachment=True)
+    return response
```

Comment:
Please end each file with a carriage return ... if only to suppress the ugly warning message.